llarp/ev/vpn.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+
+#include <net/ip_range.hpp>
+#include <net/ip_packet.hpp>
+#include <set>
+
+namespace llarp
+{
+  struct Context;
+}
+
+namespace llarp::vpn
+{
+  struct InterfaceAddress
+  {
+    constexpr InterfaceAddress(IPRange r, int f = AF_INET) : range{std::move(r)}, fam{f}
+    {}
```

Comment:
prob put a newline after this line with `{}`.  tripped me up while reading it for whatever reason.